src/sentry/models/sentryappinstallation.py

```diff
@@ -46,6 +51,12 @@ class SentryAppInstallation(ParanoidModel):
     uuid = models.CharField(max_length=64,
                             default=default_uuid)
 
+    status = BoundedPositiveIntegerField(
+        default=SentryAppInstallationStatus.PENDING,
```

Comment:
@mnoble What will this do to existing `SentryAppInstallation` objects?

Reply:
Sets everything to `PENDING` for the time being. This attribute isn't used for anything yet, so it's fine. We'll backfill the appropriate values when we're ready to actually use this attribute.